sdcclient/_scanning.py

```diff
@@ -350,6 +350,24 @@ def get_anchore_users_account(self):
 
         return [True, res.json()]
 
+    def get_image_scan_result_by_id(self, image_id, full_tag_name):
+        '''**Description**
+            Get the anchore image scan result for an image id.
+
+        **Arguments**
+            - image_id : Docker image id of the image whose scan result is to be fetched
+            - full_tag_name : The complete tag name of the image for e.g. docker.io/alpine:3.10
+
+        **Success Return Value**
+            A JSON object containing pass/fail status of image scan policy.
+        '''
+        url = self.url + "/api/scanning/v1/anchore/images/by_id/" + str(image_id) + "/check?tag=" + str(full_tag_name) + "&detail=false"
```

Comment:
It's nice to use a format string, like https://github.com/draios/python-sdc-client/pull/103/files#diff-b7f06877dadbc7c6c5e53cde465bae46R298-R302.